cmd/sliexporter.go

```diff
@@ -94,6 +97,20 @@ func (s *sliProber) executeSLIProber(cmd *cobra.Command, _ []string) error {
 			return err
 		}
 	}
+
+	if s.enableVSHNMinio || true {
```

Comment:
Is the `|| true` some debugging leftover? With this we can never disable the minio prober